cgmanifest.json

```diff
@@ -1,461 +1,641 @@
-            },
-            "type": "git"
-         }
-      },
-      {
-         "component": {
-            "git": {
-               "commitHash": "9245d481eb3e890f708ff2d7dadf2a10c04748ba",
-               "repositoryUrl": "https://github.com/llvm-mirror/libcxxabi.git"
-            },
-            "type": "git"
-         }
-      },
-      {
-         "component": {
-            "git": {
-               "commitHash": "9ce4a77f61c134bbed28bfd5be5cd7dc0e80f5e3",
-               "repositoryUrl": "https://github.com/google/upb.git"
-            },
-            "type": "git"
-         }
-      },
-      {
-         "component": {
-            "git": {
-              "commitHash": "e8c599bca6c56c44b6730ad93f6abbc9ecd60fc1",
-              "repositoryUrl": "https://github.com/microsoft/wil"
-            },
-            "type": "git"
-          }
-      },
-      {
-         "component":{
-            "type": "other",
-            "Other": {
-               "Name": "Go",
-               "Version": "1.12.6",
-               "DownloadUrl": "https://dl.google.com/go/go1.12.6.linux-amd64.tar.gz"
-            }
-         }
-      },
-      {
-         "component": {
-            "git": {
-               "commitHash": "cc4bed2d74f7c8717e31f9579214ab52a9c9c610",
-               "repositoryUrl": "https://github.com/abseil/abseil-cpp"
-            },
-            "type": "git",
-            "comments": "used by onnxruntime server"
-         }
-      },
-      {
-         "component": {
-            "type": "git",
-            "git": {
-               "commitHash": "256be563447a315f2a7993ec669460ba475fa86a",
-               "repositoryUrl": "https://github.com/abseil/abseil-cpp"
-            },
-            "comments": "used by onnxruntime"
-         }
-      },
-      {
-         "component": {
-            "git": {
-               "commitHash": "c3cceac115c072fb63df1836ff46d8c60d9eb304",
-               "repositoryUrl": "https://github.com/NVlabs/cub.git"
-            },
-            "type": "git"
-         }
-      },
-      {
-         "component": {
-            "git": {
-               "commitHash": "d98bf0278d6f59a58271425963a8422ff48fe249",
-               "repositoryUrl": "https://github.com/nlohmann/json.git"
-            },
-            "type": "git"
-         }
-      },
-      {
-         "component": {
-            "git": {
-               "commitHash": "3f0f9802553944b75015aad098d856b2d17220df",
-               "repositoryUrl": "https://github.com/microsoft/FeaturizersLibrary.git"
-            },
-            "type": "git"
-         }
-      }
-   ],
-   "Version": 1
+    "Registrations": [
+        {
+            "component": {
+                "git": {
+                    "commitHash": "1b543d6e5073b56be214394890c9193979a3d7e1",
+                    "repositoryUrl": "https://chromium.googlesource.com/chromium/llvm-project/llvm/lib/Fuzzer"
+                },
+                "type": "git"
+            }
```

Comment:
What is the reason for changing the ordering of everything and having such a large diff? 
Did the content change at all besides the re-ordering?

Reply:
Content was updated and there's more submodules than before. I don't know how the old one was generated, this is the order generated by the `get_submodules.py` script